Scripts/extras/pull_and_build_from_git.py

```diff
@@ -919,6 +932,9 @@ def execute(self):
             # Build the package
             self.build_for_platform()
 
+            # Copy extra files specified in the build config
+            self.copy_extra_files()
+
```

Comment:
Should this really be in the "if" statement?  Feels like it should happen regardless of how it gets the build.

Reply:
Done!